### PR TITLE
Fix PHP Coding Standards errors

### PR DESCRIPTION
**Pull request by Ismail-elkorchi**
This PR is a follow up of https://github.com/Automattic/wp-super-cache/pull/333.

This PR has to do only with the plugins. To ease the review process, I pushed in each commit the changes made to one plugin, except the last commit, in which I fixed some errors that I discovered while I reviewed the changes I made.

### CODE REVIEW

File: plugins/searchengine.php [hunk 1]
```diff
@@ -3,30 +3,33 @@ function wp_supercache_searchengine( $string ) {
...
+	if( '' !== $string ) {
 		return $string;
+	}
 
-	if( isset( $_COOKIE[ '7a1254cba80da02d5478d91cfd0a873a' ] ) && $_COOKIE[ '7a1254cba80da02d5478d91cfd0a873a' ] == 1 ) {
+	if ( isset( $_COOKIE['7a1254cba80da02d5478d91cfd0a873a'] ) && '1' === $_COOKIE['7a1254cba80da02d5478d91cfd0a873a'] ) {
 		$string = 'searchengine';
-	} elseif( isset( $_SERVER[ 'HTTP_REFERER' ] ) && $_SERVER[ 'HTTP_REFERER' ] != '' ) {
-		if( is_array( $passingthrough ) == false )
+	} elseif ( isset( $_SERVER['HTTP_REFERER'] ) && '' === $_SERVER['HTTP_REFERER'] ) {
```
Comment (donnchawp) [inline plugins/searchengine.php]: This should be "!==" instead of "===" I think.

File: plugins/wptouch.php [hunk 2]
```diff
@@ -3,35 +3,36 @@
...
+		if ( $cache_wptouch ) {
+			$status = __( 'enabled', 'wp-super-cache' );
+		} else {
+			$status = __( 'disabled', 'wp-super-cache' );
 		}
-	echo '<div class="submit"><input class="button-primary" ' . SUBMITDISABLED . 'type="submit" value="' . __( 'Update', 'wp-super-cache' ) . '" /></div>';
-	wp_nonce_field('wp-cache');
+		echo '<p><strong>' . sprintf( __( 'WPTouch support is now %s', 'wp-super-cache' ), $status ) . '</strong></p>';
+	}
+		echo '<div class="submit"><input class="button-primary" ' . SUBMITDISABLED . 'type="submit" value="' . __( 'Update', 'wp-super-cache' ) . '" /></div>';
```
Comment (donnchawp) [inline plugins/wptouch.php]: Should these two lines be indented one tab less?

File: plugins/wptouch.php [hunk 3]
```diff
@@ -41,36 +42,42 @@ function wp_super_cache_wptouch_admin() {
...
+	if ( '1' !== $cache_wptouch ) {
 		return false;
+	}
 
-	if ( ( isset( $_COOKIE[ 'wptouch_switch_toggle' ] ) && $_COOKIE[ 'wptouch_switch_toggle' ] == 'normal' ) ||
-		( isset( $_COOKIE[ 'wptouch-pro-view' ] ) && $_COOKIE[ 'wptouch-pro-view' ] == 'desktop' ) )
-		return true;
+	if ( ( isset( $_COOKIE['wptouch_switch_toggle'] ) && 'normal' === $_COOKIE['wptouch_switch_toggle'] ) ||
+		( isset( $_COOKIE['wptouch-pro-view'] ) && 'desktop' === $_COOKIE['wptouch-pro-view'] ) ) {
+			return true;
```
Comment (donnchawp) [inline plugins/wptouch.php]: Indented one tab too much?

**Comment by donnchawp** [comment]
Looks great, except for a few small things I've commented on above. What d'you think?

**Comment by Ismail-elkorchi** [comment]
@donnchawp I think that all the suggestions you made make sens. :smiley: I adjusted the code based on your feedback and added some small fixes.

**Pull request merged**